apps/docs/js-interpreter.md

```diff
@@ -112,6 +112,12 @@ extra work to setup your development environment. Assuming you already have the
    
 ### Step 2: Test the upgrade against the official ECMAScript test suite:
 
+> ## Warning
```

Comment:
Do folks still use this document in general? It's not clear to me how much of it is still relevant; I considered deleting the entire file, but erred on the side of keeping documentation around. More than happy to delete this entirely if it's not useful, though.

Reply:
I did not know this existed 😅 however, at least the "Component Overview" section seems relevant since we still use the JSInterpreter and CustomMarshalingInterpreter classes today.

Reply:
Lol, yeah, I suspect you're not alone.

Great, thanks! I'll leave the rest of this document as-is for now, then.